install_config/aggregate_logging.adoc

```diff
@@ -1011,6 +1012,121 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
   done
 ----
 
+[[aggregated-logging-hostPath-Elasticsearch]]
+===== Configuring hostPath storage for Elasticsearch =====
+
+You can provision {product-title} clusters using hostPath storage for Elasticsearch.
+
+To use a local disk volume on each node host as storage for an Elasticsearch replica:
+
+. Create a local mount point on each infrastructure node for the local Elasticsearch storage:
++
+----
+mkdir /usr/local/es-storage
+----
+
+. Create a filesystem on the Elasticsearch volume:
++
+----
+mkfs.ext4 /dev/xxx
+----
+
+. Mount the elasticsearch volume:
++
+----
+mount /dev/xxx /usr/local/es-storage
+----
+
+. Add the following line to `/etc/fstab`:
++
+----
+/dev/xxx /usr/local/es-storage ext4
+----
+
+. Change ownership for the mount point:
++
+----
+chown 1000:1000 /usr/local/es-storage
```

Comment:
Similar to above, add a $ in front of the command.

Reply:
Added. Thanks.